src/components/Trailer/Trailer.js

```diff
@@ -0,0 +1,17 @@
+
+export const TrailerMovie = ( videoUrl ) => {
+    const template =  `
+        <video
+        id="my-video"
+        class="video-js vjs-default-skin vjs-big-play-centered"
+        controls
+        preload="auto"
+        width="640"
```

Comment:
This should be responsive I think

Reply:
i change to width="100%" and height="auto"